sdk/digitaltwins/azure-digitaltwins-core/azure/digitaltwins/core/aio/__init__.py

```diff
@@ -5,7 +5,3 @@
 # Code generated by Microsoft (R) AutoRest Code Generator.
 # Changes may cause incorrect behavior and will be lost if the code is regenerated.
 # --------------------------------------------------------------------------
-
-from ._digitaltwins_client_async import DigitalTwinsClient
-
-__all__ = ['DigitalTwinsClient']
```

Comment:
This shouldn't have been removed - the client still needs to be exported in the `aio` namespace.

Reply:
Fiexed.